AcadeMeData/tests/test_msgtags.py

```diff
@@ -0,0 +1,19 @@
+import pytest
+from AcadeMeData.models import MessageTags, Messages, User
+
+
+@pytest.mark.django_db
+class TestMessageTagsModel:
+    @pytest.fixture
+    def user_example(self):
+        user_data = {'username': "username2212", 'password': "password", 'email': "user@example.com", 'type': "S",
+                     'university': "RU",
+                     'degree': "CS"}
+        user = User.create_user(*user_data)
```

Comment:
try this 

have it external to the class will work also

@pytest.fixture
def user_example():
    user_data = {'username': "username2212", 'password': "password", 'email': "user@example.com", 'type': "S",
                 'university': "RU",
                 'degree': "CS"}
    user = User.create_user(*user_data)
    return user